packages/common-all/src/fuse.ts

```diff
@@ -132,6 +95,18 @@ export type SerializedFuseIndex = ReturnType<
 
 type FuseEngineOpts = {
   mode?: DEngineMode;
+  /** If specified must be within 0-1 range. */
+  fuzzThreshold: number;
+};
+
+export const getThresholdValue = (configThreshold: number) => {
```

Comment:
nitpick: for this, can we rename to `getCleanThresholdValue` to indicate that this value is sanitized/validated?